### PR TITLE
Updated README with example usage of new cost computing fn

### PR DESCRIPTION
**Pull request by mturk24**
Can see what README changes look like here: https://github.com/mturk24/yolopandas/tree/add-token-estimation-per-query

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,6 +46,24 @@ df.llm.query("What item is the least expensive?", yolo=True)
 df.llm.query("Group by type and take the mean of all numeric columns.", yolo=True).llm.query("Make a bar plot of the result and use a log scale.", yolo=True)
 ```
 
+Also, if you want to get a better idea of how much each query costs, you can use the function `run_query_with_cost` found in the utils package to compute the cost in $USD broken down by prompt/completion tokens:
+
+```python
+
+from utils.query_helpers import run_query_with_cost
```
Comment (ccurme) [inline README.md]: Shouldn't this be
```suggestion
from yolopandas.utils.query_helpers import run_query_with_cost
```

File: README.md [hunk 2]
```diff
@@ -46,6 +46,24 @@ df.llm.query("What item is the least expensive?", yolo=True)
 df.llm.query("Group by type and take the mean of all numeric columns.", yolo=True).llm.query("Make a bar plot of the result and use a log scale.", yolo=True)
 ```
 
+Also, if you want to get a better idea of how much each query costs, you can use the function `run_query_with_cost` found in the utils package to compute the cost in $USD broken down by prompt/completion tokens:
```
Comment (ccurme) [inline README.md]: nitpick:
```suggestion
Also, if you want to get a better idea of how much each query costs, you can use the function `run_query_with_cost` found in the utils module to compute the cost in $USD broken down by prompt/completion tokens:
```

**Review by ccurme — CHANGES_REQUESTED** [review]
Thanks for this. Can you update the import statement?

btw, `utils.query_helpers` was released in yolopandas==0.0.6.

**Comment by mturk24** [comment]
> Thanks for this. Can you update the import statement?
> 
> btw, `utils.query_helpers` was released in yolopandas==0.0.6.

Just fixed with new commits thank you! And awesome :)

**Review by ccurme — APPROVED** [review]

**Pull request merged**